ets-ecole-de-technologie-superieure.csl

```diff
@@ -88,23 +81,21 @@
         </group>
       </if>
       <else>
-        <text variable="container-title" form="short" font-style="italic" suffix=", "/>
+        <text variable="container-title" form="short" font-style="italic"/>
       </else>
     </choose>
   </macro>
   <macro name="title">
-    <group delimiter=" ">
-      <choose>
-        <if type="chapter paper-conference article-journal article-magazine article-newspaper webpage thesis" match="none">
-          <text variable="title" font-style="italic"/>
-        </if>
-        <else>
-          <group>
-            <text variable="title" quotes="true"/>
-          </group>
-        </else>
-      </choose>
-    </group>
+    <choose>
+      <if type="chapter paper-conference article-journal article-magazine article-newspaper webpage post post-weblog thesis" match="none">
+        <text variable="title" font-style="italic"/>
+      </if>
+      <else>
+        <group>
+          <text variable="title" prefix="« " suffix=" »"/>
```

Comment:
my understand is that that's been fixed for the next release (and would have likely also affected hard-coded guillemets):
https://forums.zotero.org/discussion/50595/problems-with-last-update-firefox-word-2010-not-starter-edition/#Item_18
So please revert that change (in general, better to report bugs on the Zotero forums than to work around them in citation styles. We're usually disinclined to accept changes to styles based on one or two misbehaving reference managers).